### PR TITLE
Translate the 'Need help' label in the _footer partial

### PR DESCRIPTION
**Pull request by tupaja**
To make the translation flexible (i.e. allowing for the email to be
placed anywhere in the sentence) email attribute is interpolated.
This required adjusting the `t` method's signature. Unfortunately the
scribble gem doesn't support arrays or hashes, so a query string was used
instead as a workaround.

### CODE REVIEW

**Comment by parisyee** [comment]
@tupaja you're also going to want to update the spec for `t`

File: spec/methods/t_spec.rb [hunk 1]
```diff
@@ -8,6 +8,11 @@ module Scribble::Methods
         expect(render("{{t('.foo')}}")).to eq "bar"
         expect(render("{{t('bar')}}")).to eq "baz"
       end
+
+      it "allows using variables" do
+        expect(render("{{t('email_me', 'email=me@example.com')}}")).to eq \
```
Comment (attack) [inline spec/methods/t_spec.rb]: how do you feel about using the standard api here?

```
t('email_me', email: 'me@example.com')
```

**Comment by rupurt** [comment]
🌴

File: spec/methods/t_spec.rb (hunk 1 above)
Comment (tupaja) [inline spec/methods/t_spec.rb]: I don't think it's possible because of scribble limitations (see my first comment above)

**Pull request merged**